components-api/src/main/scala/pl/touk/nussknacker/engine/api/properties/ScenarioProperties.scala

```diff
@@ -0,0 +1,25 @@
+package pl.touk.nussknacker.engine.api.properties
+
+import io.circe.generic.JsonCodec
+import pl.touk.nussknacker.engine.api.component.SingleScenarioPropertyConfig
+
+@JsonCodec case class ScenarioProperties(
+    propertiesConfig: Map[String, SingleScenarioPropertyConfig],
+    docsUrl: Option[String] = None
+) {
+
+  // it will overwrite docsUrl, with the approach that the latest config is the proper one.
+  def ++(newConfig: ScenarioProperties): ScenarioProperties = {
```

Comment:
I doesn't see a reason to have this confusing method. In the only place that you use it, you merge only propertiesConfig map

Reply:
Ok, removed and changed to mergeConfigs in ProcessingTypeData